src/routes/auth/index.js

```diff
@@ -0,0 +1,39 @@
+import express  from 'express';
+import cookieParser from 'cookie-parser';
+import session  from 'express-session';
+import SequelizeStoreFactory from 'connect-session-sequelize';
+
+import basic    from './basic';
+import facebook from './facebook';
+
+import config   from '../../config';
+import { sequelize }       from '../../models/db';
+
+const router = express.Router();
+
+router.use('/basic', basic);
+
+// initalize sequelize with session store
+const SequelizeStore = SequelizeStoreFactory(session.Store);
+const isProduction = process.env.NODE_ENV === 'production';
+router.use(cookieParser());
+router.use(session({
+  secret: 'keyboard cat',
+  store: new SequelizeStore({
+    db: sequelize
+  }),
+  cookie: {
+    path: `/${config.get('version')}/auth`,
+    secure: isProduction,
+  },
+  name: 'connect.sid.auth',
+  proxy: isProduction, // TODO maybe better configure this in config
```

Comment:
this is when node is in HTTP but there is a proxy in HTTPS, when we use `secure: true` above